applications/common/backend/src/test/java/pl/ds/kyanite/common/components/models/table/TableComponentTest.java

```diff
@@ -37,25 +44,79 @@ public class TableComponentTest {
   @BeforeEach
   public void init() {
     context.addModelsForClasses(TableComponent.class);
-    context.load().json(requireNonNull(
-        Thread.currentThread().getContextClassLoader().getResourceAsStream("table.json")), PATH);
```

Comment:
A new test appeared in this class.
So those two lines are not common for all tests anymore - move to //given sections of the test that use this